addons/point_of_sale/static/src/css/pos_receipts.css

```diff
@@ -1,7 +1,8 @@
 .pos-receipt-print {
-    width: 512px;
-    font-size: 27px;
+    width: 400px;
+    font-size: 18px;
```

Comment:
Why do you reduce the size and font-size of the ticket ?

Reply:
![image](https://github.com/odoo/odoo/assets/157003503/4c5a687a-66a0-4ef5-a7e1-ccc68acef662)

Because of the large font size, it becomes so messy for a large amount.
After changing only the font size, the image width becomes like this.
![image](https://github.com/odoo/odoo/assets/157003503/27ee7407-ebb9-476d-802b-cccf2b9ae9b2)

So I changed the width according to font size.